github/helpers/githubSDK.ts

```diff
@@ -631,14 +647,12 @@ export async function getPullRequestComments(
 export async function getIssuesComments(
     http: IHttp,
     repoName: string,
-    access_token: string,
     issueNumber: string | number
 ) {
     const response = await http.get(
         `https://api.github.com/repos/${repoName}/issues/${issueNumber}/comments`,
         {
             headers: {
-                Authorization: `token ${access_token}`,
                 "Content-Type": "application/json",
```

Comment:
@ayush3160 Removing the access token itself is not the solution to the app breaking for non authorized users. There should be conditional handling. You should always try finding a solution to the problem instead of limiting the problem based on your solution. 
Think of this way : 
`how do I make this feature a great user experience for every user ?`
Cases : 
- User is not logged in: 
    - They should  be able to see comments from a public repository. 
    - They should not see an option to comment since they are not logged in. But they should be given a hint to login if they wish to comment.
- The user is logged : 
    - They should  be able to see comments from a public repository. and private repositories.  
    - They should be able to a add comments to issues on both public and private repositories.